elastic/kibana
# [Fleet] Use new dataView API for agent logs filters

Closes https://github.com/elastic/kibana/issues/135970

## Summary

Use `dataView` API in agent dataset and logs filters. These two components were composing their own dataView-like objects. Now they compose `DataViewSpecs` and instantiate them with `DataViews.create`.
 
## Testing steps

 Run an agent, navigate to agent logs and verify that the "dataset" and "log level" filter work as usual:

<img width="812" alt="Screenshot 2023-02-14 at 17 11 38" src="https://user-images.githubusercontent.com/16084106/218794511-bd2d2124-76b8-4972-90d2-5d7703cf682a.png">
<img width="920" alt="Screenshot 2023-02-14 at 17 11 30" src="https://user-images.githubusercontent.com/16084106/218794532-c8449a7a-d0ed-40cd-bc79-de716f0178d2.png">



### Checklist
